src/components/Header.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components/macro'
+import HeaderButton from './HeaderButton'
+
+export default function Header({ appState, handleClick }) {
+  const color =
+    appState === 'default'
+      ? 'var(--default-heder-color)'
```

Comment:
```suggestion
      ? 'var(--default-header-color)'
```
 Shouldn't work like that if the color variable is written wrong 🎨

Reply:
😱